src/Compilers/CSharp/Portable/Binder/Binder.cs

```diff
@@ -687,7 +687,7 @@ internal BoundStatement WrapWithVariablesIfAny(CSharpSyntaxNode scopeDesignator,
             }
 
             return new BoundBlock(statement.Syntax, locals, ImmutableArray.Create(statement))
-                { WasCompilerGenerated = true };
+            { WasCompilerGenerated = true };
```

Comment:
This may be a degradation.

Reply:
It's not typical to indent the open brace for an object initializer, is it?

Reply:
This seems like the desired behavior to me. If someone disagrees, please feel free to create an issue. 😄